<?xml version="1.0"?>
<launch>
    <arg name="object_name" default="box1"/>
    <arg name="force" default="0.0153"/>
    <arg name="front_length" default="0.057"/>
    <arg name="place_x" default="0.2"/>
    <arg name="place_y" default="0.1"/>
    
    
    <node pkg="arm_move" type="arm_move_main_1" name="arm_move_main_1">
        <param name="object_name" value="$(arg object_name)"/>
        <param name="force" value="$(arg force)"/>
        <param name="front_length" value="$(arg front_length)"/>
        <param name="place_x" value="$(arg place_x)"/>
        <param name="place_y" value="$(arg place_y)"/>
    </node>
</launch>